packages/alfa-css/src/value/keyword.ts

```diff
@@ -72,7 +74,7 @@ export namespace Keyword {
         // `Keyword<"foo"> | Keyword<"bar">`, not `Keyword<"foo" | "bar">`. Why?
         // Because the former is assignable to the latter, but the latter isn't
         // assignable to the former.
-        Keyword.of(ident.value.toLowerCase()) as { [K in T]: Keyword<K> }[T]
+        Keyword.of(ident.value.toLowerCase()) as ToKeyword<T>
```

Comment:
I'm not too keen on this as the difference between `Keyword<"foo" | "bar">` and `Keyword.ToKeyword<"foo" | "bar">` is too small, especially when broken over several lines. The former type is incorrect and the latter will result in a correct type being produced, but all it takes is forgetting `.ToKeyword` for the type to end up wrong. It's much easier to tell `Keyword<"foo"> | Keyword<"bar">` from `Keyword<"foo" | "bar">`, even if it does cost some additional keystrokes. Correctness over conciseness though 😄